assets/eip-7265/test/core/user/CircuitBreakerUserOps.t.sol

```diff
@@ -0,0 +1,362 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Assuming you have the authority to change the license, we require a more permissive license like:

```suggestion
// SPDX-License-Identifier: CC0-1.0
```

If you do not have authority to change the license, please remove this file.